app/controllers/applications_controller.rb

```diff
@@ -0,0 +1,41 @@
+class ApplicationsController < ApplicationController
+  def index
+  	@applications = Application.all
+  end
+
+  def new
+  	@application = Application.new
+  end
+
+  def create
+  	application = Application.new(application_params)
+  	if application.save
+      redirect_to applications_path
+    else
+      error_response(application)
+    end
+  end
+
+  def show
+  	@application = Application.find(params[:id])
+  	@info = @application.info
+  	@application_tenant = @info.tenant
+  	@property = @application.property
+  end
+
+  def destroy
+  	@application = Application.find(params[:id])
+		@application.destroy
```

Comment:
should have a check for if the destroy went through

https://github.com/calblueprint/RMI/blob/81db886779a48fd51cef6b5673203bf28d1b5fd3/app/controllers/api/answers_controller.rb#L51